pkg/instance/storage.go

```diff
@@ -130,6 +130,9 @@ func (s *storage) AddFileBytes(bytes []byte, dest string, chown string) error {
 	if _, err := tmpfile.Write(bytes); err != nil {
 		return err
 	}
+	if err := tmpfile.Chmod(0644); err != nil {
```

Comment:
[nit] would be nicer to extract the `0644` constant

Reply:
thank you!
https://github.com/celestiaorg/knuu/pull/589/commits/34a7443782f68347cc7e267ec868a0ee94e8d5f5